build/helm/keylime/charts/keylime-agent/templates/daemonset.yaml

```diff
@@ -107,6 +108,9 @@ spec:
               mountPath: /var/lib/keylime-persistent
             - name: secure
               mountPath: /var/lib/keylime/secure
+              readOnly: true
+            - name: tmpfs
+              mountPath: /tmp
```

Comment:
I assume this is because you discovered that the agent needs a writable temp directory at some point?

Reply:
Correct. The agent was conking out w/o a writable `/tmp`, so I added it.